libkbfs/backpressure_disk_limiter.go

```diff
@@ -470,6 +479,49 @@ func (bdl *backpressureDiskLimiter) onBlocksDelete(
 	bdl.fileTracker.onBlocksDelete(blockFiles)
 }
 
+func (bdl *backpressureDiskLimiter) onDiskBlockCacheDelete(
+	ctx context.Context, blockBytes int64) {
+	if blockBytes == 0 {
+		return
+	}
+	bdl.lock.Lock()
+	defer bdl.lock.Unlock()
+	bdl.byteTracker.onBlocksDelete(blockBytes)
+}
+
+func (bdl *backpressureDiskLimiter) beforeDiskBlockCachePut(
+	ctx context.Context, blockBytes, diskBlockCacheBytes int64) (
+	availableBytes int64, err error) {
```

Comment:
I think we also need an `afterDiskBlockCachePut`, since otherwise if the cache put fails somehow, we'd 'leak' those bytes. And if it succeeds we want to do something similar to `afterBlockPut` (i.e., update `bt.used`).